gbpservice/neutron/config/nsf.ini

```diff
@@ -0,0 +1,13 @@
+[DEFAULT]
+debug=True
+rabbit_password=openstack 
+rabbit_userid=stackrabbit
+rabbit_hosts=<openstack controller ip adress:rabbitPort>  #Example 192.168.2.107:5672
+rabbit_port=5672
+rabbit_host=<openstack controller ip address>
```

Comment:
Can these be derived from existing ini files?
rabiit_hosts contain the same info as rabbit_port, rabbit_host.